activestorage/app/models/active_storage/blob.rb

```diff
@@ -116,6 +116,14 @@ def create_before_direct_upload!(key: nil, filename:, byte_size:, checksum:, con
     def generate_unique_secure_token(length: MINIMUM_TOKEN_LENGTH)
```

Comment:
THis is great work did you finish building this feature?

Reply:
Yes I will try, now I have more time.

Reply:
@Ispirett the PR have been pushed to rails. Do not hesitate to upvote!